_includes/payment-order-other-features.md

```diff
@@ -1392,6 +1392,7 @@ elements. When in doubt, please follow the [robustness principle](https://en.wik
 [one-click-payments]: /payment-menu/other-features#one-click-payments
 [split-settlement]: /payment-menu/other-features#split-settlement
 [settlement-and-reconciliation]: /payment-menu/other-features#settlement-and-reconciliation
+[recur]: /payment-menu/other-features#recurring-payments
```

Comment:
I don't think you can use use `payment-menu` in the URL since this include is also used in Checkout?